common/src/main/kotlin/de/hpi/cloud/common/utils/String.kt

```diff
@@ -1,18 +1,31 @@
 package de.hpi.cloud.common.utils
 
+import de.hpi.cloud.common.types.L10n
+
+fun String.trimNotEmpty(): String = trim().also {
+    require(it.isNotEmpty()) { "String must not be blank" }
+}
+
+fun L10n<String>.trimNotEmpty(): L10n<String> =
+    L10n(
+        values.mapValues { (locale, value) ->
+            value.trim().also {
```

Comment:
give trimNotEmpty a message parameter and reuse the method :)

Reply:
These two utility methods were for an early validation prototyping but are currently unused, hence I remove them for now